src/reaper_osara.cpp

```diff
@@ -2589,15 +2881,11 @@ bool maybeFixTabInSaveDialog(bool previous) {
 }
 
 // Handle keyboard keys which can't be bound to actions.
-// REAPER's "accelerator" hook isn't enough because it doesn't get called in some windows.
+// REAPER's "accelerator" hook isn't enough because it doesn't get called in
+// some windows.
 LRESULT CALLBACK keyboardHookProc(int code, WPARAM wParam, LPARAM lParam) {
 	const bool isKeyDown = !(lParam & 0x80000000);
-	if (!isKeyDown || code != HC_ACTION || (
-		wParam != VK_APPS && wParam != VK_CONTROL &&
-				wParam != VK_F10 && wParam != VK_RETURN &&
-				wParam != VK_F6 && wParam != 'B' &&
-				wParam != VK_TAB && wParam != VK_DOWN &&
-				wParam != VK_UP && wParam != VK_SPACE)) {
+	if (!isKeyDown || code != HC_ACTION || (wParam != VK_APPS && wParam != VK_CONTROL && wParam != VK_F10 && wParam != VK_RETURN && wParam != VK_F6 && wParam != 'B' && wParam != VK_TAB && wParam != VK_DOWN && wParam != VK_UP && wParam != VK_SPACE)) {
```

Comment:
What the actual hell? What happened to the maximum line length? This actually makes this much harder to read.

Reply:
Wow this is extremely weird indeed.

Reply:
I actually fixed continuation indent and now it seems to break up properly again.

Reply:
It now adds breaks again, but insists to add an extra space to the second line:

```
	if (!isKeyDown || code != HC_ACTION ||
			(wParam != VK_APPS && wParam != VK_CONTROL && wParam != VK_F10 && wParam != VK_RETURN && wParam != VK_F6 &&
			 wParam != 'B' && wParam != VK_TAB && wParam != VK_DOWN && wParam != VK_UP && wParam != VK_SPACE)) {

```

I'm utterly confused!